online-unsafe-recovery.md

```diff
@@ -161,12 +170,44 @@ PD 下发恢复计划后，会等待 TiKV 上报执行的结果。如上述输
 
 ### 第 3 步：检查数据索引一致性（RawKV 不需要）
 
-执行完成后，可能会导致数据索引不一致。请使用 SQL 的 [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md)、`ADMIN RECOVER`、`ADMIN CLEANUP` 命令对受影响的表（从 `"Unsafe recovery finished"` 输出的 `"Affected table ids"` 可知）进行数据索引的一致性检查及恢复。
-
 > **注意：**
 >
 > 数据可以读写并不代表没有数据丢失。
 
+执行完成后，数据和索引可能会不一致。请使用 SQL 的 [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) 对受影响的表进行数据索引一致性检查。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECK TABLE table_name;
+```
+
+若结果有不一致的索引，可以通过重命名旧索引，创建新索引，删除旧索引的方式来修复数据索引不一致的问题。
+
+重命名旧索引：
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name RENAME INDEX index_name TO index_name_lame_duck;
+```
+
+创建新索引：
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name ADD INDEX index_name (column_name);
+```
+
+删除旧索引：
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name DROP INDEX index_name_lame_duck;
+```
+
```

Comment:
````suggestion
若结果有不一致的索引，可以通过重命名旧索引、创建新索引、删除旧索引的方式来修复数据索引不一致的问题。

重命名旧索引：

```sql
ALTER TABLE table_name RENAME INDEX index_name TO index_name_lame_duck;
```

创建新索引：

```sql
ALTER TABLE table_name ADD INDEX index_name (column_name);
```

删除旧索引：

```sql
ALTER TABLE table_name DROP INDEX index_name_lame_duck;
```

````